File: src/python/talk.py
# -*- coding: utf-8 -*-
#! /usr/bin/env python
#use python2.7 lang.
#and set env like this.
#export A3RT_API_KEY_Talk="XXXXXXXXXXXXXX"

import os
import json
import string
import codecs
import requests

def getAppInfo(sApiKey, text):

    sURL = "https://api.a3rt.recruit-tech.co.jp/talk/v1/smalltalk"
    params = {'apikey' : sApiKey,'query' : text}
    stRes = requests.post(sURL,data=params)

    return stRes

if __name__ == '__main__':
	sApiKey = os.environ.get('A3RT_API_KEY_Talk')
	text = "馬が走っています。"
	stRes = getAppInfo(sApiKey, text)

	data = json.loads(stRes.text)
	print(data)

File: src/php/talk.php
<?php
//php implimentation
//and set env like this.
//export A3RT_API_KEY_Talk="XXXXXXXXXXXXXX"

use GuzzleHttp\Client;
require "vendor/autoload.php";

$sApiKey = getenv('A3RT_API_KEY_Talk');
$text = "馬が走っています。";
$base_url = 'https://api.a3rt.recruit-tech.co.jp/talk/v1/smalltalk';
$query = ['apikey'=>$sApiKey,'query'=>$text]; 
print var_export($query , true);

$client = new Client();
$res = $client->post($base_url, $query);

$response_body = $res->getBody();
$response_body = json_decode($response_body, true);

print var_export($response_body , true);


?>
